src/settings-ui/Settings.UI/SettingsXAML/Controls/ColorFormatEditor.xaml

```diff
@@ -101,9 +96,8 @@
 
             <TextBlock
```

Comment:
Looking at this, I wonder if we can make these titles less verbose?

"The following parameters:" => "Parameters"

"The red, green, blue and alpha values can be formatted to the following formats" => "Formatting parameters"

"Example: %ReX means red value in hex uppercase two digits format" => "Example: %ReX = ff"

Probably there are better ways to describe these :)

Reply:
Noted